juno-1/MONETA_UPGRADE.md

```diff
@@ -36,6 +36,10 @@ $DAEMON_HOME/cosmovisor/upgrades/moneta/bin/junod version
 
 Fees are set by each individual validator. The ability to spam TXs (Smart Contracts or otherwise) is a potential attack vector for the Juno network, so if you haven't already set minimum gas prices, please change these in `.juno/config/app.toml`. We suggest `0.025ujuno`.
 
+```
+perl -i -pe 's/^minimum-gas-prices = .+?$/minimum-gas-prices = "0.025ujuno"/' ~/.juno/config/app.toml
```

Comment:
i have a soft spot in my heart for perl, but to rhyme with the other instructions in docs.junonetwork.io, how about sed?

```suggestion
```
perl -i -pe 's/^minimum-gas-prices = .+?$/minimum-gas-prices = "0.025ujuno"/' ~/.juno/config/app.toml
```
sed -i.bak -e "s/^minimum-gas-prices *=.*/minimum-gas-prices= \"0.025ujuno\"/" ~/.juno/config/config.toml

```

Reply:
Haha, sure, I have no strong opinions on it. Thanks for weighing in.

Reply:
Huh! Juno does tend towards `sed`:

https://docs.junonetwork.io/validators/joining-mainnet#set-persistent-peers-1

The more you know 🤷